examples/subplots_axes_and_figures/ganged_plots.py

```diff
@@ -0,0 +1,40 @@
+"""
+==========================
+Creating adjacent subplots
+==========================
+
+To create plots that share a common axes (visually) you can set the
+hspace between the subplots to zero. Passing sharex=True when creating the
+subplots will automatically turn off all x ticks and labels except those on the
+bottom axis.
+
+In this example the plots share a common xaxis but you can follow the
```

Comment:
x axis (with a space) (or at least be consistent just below when you write y axis)